### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by Sourpusss**
The most recent file accommodates all of emailboxu's edits.

### CODE REVIEW

**Comment by emailboxu** [comment]
It's actually "When ground up, this herb releases a strong smell that attracts monsters. If this is then slathered on things (((goods, etc.))), monsters will protect it like their own nest."

File: ITEM_en.tsv [hunk 1]
```diff
@@ -4681,7 +4681,7 @@ ITEM_20150428_005067	The third revelation found in the Thorn Forest. It's readab
 ITEM_20150428_005068	The fourth revelation found in the Royal Tomb. It's readable.
 ITEM_20150428_005069	The sixth revelation found in the Cathedral. It's readable.
 ITEM_20150428_005070	There are many rumors that can hear the voice of the King at the Royal Tomb Chapel.
-ITEM_20150428_005071	Reagents of testing soul that Graschius passed.
+ITEM_20150428_005071	Reagents of testing Graschius's soul.
```
Comment (emailboxu) [inline ITEM_en.tsv]: Reagents for

File: ITEM_en.tsv [hunk 2]
```diff
@@ -4830,15 +4830,15 @@ ITEM_20150612_005216	A propaganda to encourage the surrender sprayed by Kingdom
...
+ITEM_20150612_005220	This will finish off the smelly monsters.  If you leave without finishing them off, they'll continue propagating the nest.
+ITEM_20150612_005221	Damaged Belongings
 ITEM_20150612_005222	Seems the trail of times. There were fully filled with the damaged items.
 ITEM_20150612_005223	Monster Ink
 ITEM_20150612_005224	Such a black liquid obtained from the monsters.
 ITEM_20150612_005225	Monster Periderm Paper
 ITEM_20150612_005226	Such a thin film can replace the paper
 ITEM_20150612_005227	Blender
-ITEM_20150612_005228	
+ITEM_20150612_005228	Throw a monster in and make some juice!
```
Comment (emailboxu) [inline ITEM_en.tsv]: "A restraining device that allows the user to hold a monster still while he or she collects its extract."

File: ITEM_en.tsv [hunk 3]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
 ITEM_20150612_005304	Green Kokateu Bone
 ITEM_20150612_005305	A Green Kakatue bons is very hard so it is mainly used to make the arrowheads.
 ITEM_20150612_005306	Colson Signal Bell
-ITEM_20150612_005307	A tool send the signal to his companion.
+ITEM_20150612_005307	A tool Colson used to send a signal to his companion.
```
Comment (emailboxu) [inline ITEM_en.tsv]: "Tools that the Hunters used to send signals to each other. Curiously, their Companions only respond to their owners' bells."

File: ITEM_en.tsv [hunk 4]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
 ITEM_20150612_005308	Scorpion Poison Sample
 ITEM_20150612_005309	An animal poison sample obtained from the scorpion.
 ITEM_20150612_005310	Thorn Poison Sample
 ITEM_20150612_005311	A poison sample obtained from the thorn tree stump.
 ITEM_20150612_005312	Light Well Poison Sample
-ITEM_20150612_005313	A land of poison sample obtained from the light well.
+ITEM_20150612_005313	Apoison sample obtained from the light well.
```
Comment (emailboxu) [inline ITEM_en.tsv]: A poison

File: ITEM_en.tsv [hunk 5]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
 ITEM_20150612_005314	Water Bowl
-ITEM_20150612_005315	A tool scoop the poison of light well
+ITEM_20150612_005315	A tool to scoop out the poison in the light well
```
Comment (emailboxu) [inline ITEM_en.tsv:4928]: "A tool used to draw out the poison in the light well."

File: ITEM_en.tsv [hunk 6]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
 ITEM_20150612_005316	Dusk Grass
-ITEM_20150612_005317	A grass that Nethercaffe like very well.
+ITEM_20150612_005317	A grass that Nethercaffe particularly enjoy.
 ITEM_20150612_005318	Chalk for display
 ITEM_20150612_005319	A tool that marks on the wall or floor.
 ITEM_20150612_005320	Wild Honey
-ITEM_20150612_005321	A honey gathered by the wild bees. It is not suitable for eating but cure purpose is very useful.
+ITEM_20150612_005321	A honey gathered by the wild bees. It is not suitable for eating but its recovery purposes are very useful.
```
Comment (emailboxu) [inline ITEM_en.tsv]: Honey gathered by wild bees. It's not suitable for as food, but its recovery effects are great.

File: ITEM_en.tsv [hunk 7]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
 ITEM_20150612_005311	A poison sample obtained from the thorn tree stump.
 ITEM_20150612_005312	Light Well Poison Sample
-ITEM_20150612_005313	A land of poison sample obtained from the light well.
+ITEM_20150612_005313	Apoison sample obtained from the light well.
 ITEM_20150612_005314	Water Bowl
-ITEM_20150612_005315	A tool scoop the poison of light well
+ITEM_20150612_005315	A tool to scoop out the poison in the light well
 ITEM_20150612_005316	Dusk Grass
-ITEM_20150612_005317	A grass that Nethercaffe like very well.
+ITEM_20150612_005317	A grass that Nethercaffe particularly enjoy.
```
Comment (emailboxu) [inline ITEM_en.tsv]: A grass that Nethecaffes (plural)

File: ITEM_en.tsv [hunk 8]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
 ITEM_20150612_005318	Chalk for display
 ITEM_20150612_005319	A tool that marks on the wall or floor.
 ITEM_20150612_005320	Wild Honey
-ITEM_20150612_005321	A honey gathered by the wild bees. It is not suitable for eating but cure purpose is very useful.
+ITEM_20150612_005321	A honey gathered by the wild bees. It is not suitable for eating but its recovery purposes are very useful.
 ITEM_20150612_005322	Nethecaffe Essence
-ITEM_20150612_005323	A nasty essence of Nethecaffe. It is good medicine to mix the herbs.
+ITEM_20150612_005323	A nasty essence of Nethecaffe. It is a good medicine to mix in the herbs.
```
Comment (emailboxu) [inline ITEM_en.tsv]: A foul smelling secretion from a Nethecaffe. When mixed with herbs it produces a good medicine.

File: ITEM_en.tsv [hunk 9]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
 ITEM_20150612_005324	Rambear Nails
-ITEM_20150612_005325	Sharp nails to use as the primary weapons of Reamber. Even it was cutted looks like alive.
-ITEM_20150612_005326	Traver Old Book
-ITEM_20150612_005327	An old book came from the traveler. Hunter Natasha maybe able to find something in this book.
-ITEM_20150612_005328	A letter stamped seal
-ITEM_20150612_005329	A letter was left behind the old book. Evonipon's trace of the Tower of the Stars was recorded.
+ITEM_20150612_005325	Sharp nails used as the primary weapons of Reamber. Even after being cut it looks alive.
```
Comment (emailboxu) [inline ITEM_en.tsv]: Even after being cut from its claws they are still wickedly sharp. (Liberal translation, the actual translation doesn't make much sense in English.)

File: ITEM_en.tsv [hunk 10]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
+ITEM_20150612_005326	Traveler's Old Book
+ITEM_20150612_005327	An old book came that from the traveler. Hunter Natasha maybe able to find something in this book.
```
Comment (emailboxu) [inline ITEM_en.tsv]: An old book that came... Hunter Natasha may be

File: ITEM_en.tsv [hunk 11]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
+ITEM_20150612_005328	A seal stamped letter
```
Comment (emailboxu) [inline ITEM_en.tsv]: Seal Stamped Letter (Item Name)

File: ITEM_en.tsv [hunk 12]
```diff
@@ -4917,28 +4917,28 @@ ITEM_20150612_005303	It was namded since it shoot up as follow the footsteps. It
...
+ITEM_20150612_005329	A letter was left behind in the old book. Evonipon's trace of the Tower of the Stars was recorded.
```
Comment (emailboxu) [inline ITEM_en.tsv:4942]: A letter that was.. The wrongful acts done by Evonipon in the Tower of the Stars are recorded in it.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**